client/src/app/page.tsx

```diff
@@ -1,4 +1,78 @@
-import { CounterText } from "@/ui/CounterText";
+"use client";
+//import { LandingNavbar } from "@/ui/LandingNavbar";
+import "./LandingPage.css";
+import Image from "next/image";
+//import { TextCarousel } from "@/ui/TextCarousel";
+import { Typography } from "@mui/material";
+import Link from "@mui/material/Link";
+
 export default function Home() {
-  return <CounterText />;
+  {
+    /*Added Image Array to allow for easy image changes*/
+  }
+  const ImageData = [
```

Comment:
nit: I would really only do this if you're iterating on the array and using it to render a particular component. Here  you're manually indexing the data on each object to render on the component. I don't see why to keep it like this including the styling of the component. I think it makes it harder to change the styles especially for styling since the developer has to go back to the array and find the index it's associated with to change the component.

Summary:
- Store metadata, not styling. Keep the styles on the component unless you're using a value from it to render content
- Only do this method too if you're iterating on the array and rendering the component, example with the map function. If you're going to manually index it, just remove the array and place values of the props

Reply:
removed image array